File: spec/factories/idea_statuses.rb
FactoryBot.define do
  factory :idea_status do
    title_multiloc {{
      "en" => "At the mayor",
      "nl-BE" => "Bij de burgemeester"
    }}
    ordering 2
    code "custom"
    color "#AABBCC"
    description_multiloc {{
      "en" => "This idea has been presented to the mayor",
      "nl-BE" => "Het idee werd voorgesteld aan de burgemeester"
    }}
  end
end

File: app/services/__mocks__/ideaStatuses.ts
export function makeStatus(id) {
  return {
    id,
    type: 'idea_status',
    attributes: {
      title_multiloc: { en: 'Just a status' },
      color: 'blue',
      code: id,
      ordering: 0,
      description_multiloc: { en: 'The status of an idea that has a status' }
    }
  };
}
